src/styles.scss

```
@@ -1,9 +1,74 @@
 /* You can add global styles to this file, and also import other style files */
+@import "src/assets/scss/mixins/mobile";
+@import "src/assets/scss/mixins/dark-theme";
+
 
 html {
 	font-size: 16px;
 }
 
 body {
 	font-family: sans-serif;
+	padding: 2rem;
 }
+
+.grid {
+	&__header-item {
+		padding: 1em;
+	    border-bottom:2px solid #F6F6F6;
+   }
+
+	&__body-item {
+		padding: 1em;
+	}
+
+	app-item {
+		&:hover{
+			div {
```

или объединяй с следующим стилем, или вот так
```suggestion
	app-item:hover div {
```